src/tests/payment/test_stripe_webhook.py

```diff
@@ -0,0 +1,371 @@
+import hashlib
+import hmac
+import json
+import time
+from datetime import datetime, timedelta
+from decimal import Decimal
+from unittest import mock
+
+import pytest
+from django.test import Client
+from django.urls import reverse_lazy
+from django.utils import timezone
+
+from payment.models import CustomerDetail, Subscription, User
+
+PASSWORD = "password1"
+FIRSTNAME = "John"
+LASTNAME = "Doe"
+EMAIL = "johndoe@example.fr"
+USERNAME = EMAIL
+EMAIL_HOST_USER = "support@ordopro.fr"
```

Comment:
DRY, look into conftest.py for merging common fixture into a "common ancestor" of your file tree, so probably just `src/tests/conftest.py`

Reply:
I refactored the code by applying the DRY (Don't Repeat Yourself) principle, centralizing common fixtures in the conftest.py file.